src/app/sections/Search.svelte

```diff
@@ -135,12 +141,12 @@ async function onKeypress(event) {
       {/if}
     </div>
     <input 
+      type="search"
       aria-autocomplete="list"
       aria-controls="telemetry-search-results" 
       on:focus={turnOnSearch}
       bind:this={inputElement}
       placeholder="search for a telemetry probe"
-      on:blur={turnOffSearch}
```

Comment:
Seems we'll no longer need the `turnOffSearch()` function above since we set the store directly.

Reply:
Yep, good call. Thank you!